EnglishFilter/sections/adservers.txt

```diff
@@ -6,6 +6,7 @@
 !! https://a.bestcontentitem.top/warp/*?r=
 !
 ||poemsdeucewee.com^
+||ad-vortex.com^
```

Comment:
We already have this rule in Easylist.

Reply:
Then that must be added after my commit.